v4/source/commands.yaml

```diff
@@ -166,6 +166,54 @@
 
             // ListAutocompleteCommands
             listCommands, resp := Client.ListAutocompleteCommands(<TEAMID>)
+  '/teams/{team_id}/commands/autocomplete_suggestions':
+    get:
+      tags:
+        - commands
+      summary: List commands' autocomplete data
+      description: |
+        List commands' autocomplete data for the team.
+        ##### Permissions
+        `view_team` for the team.
```

Comment:
Please add the minimum server version similar to https://github.com/mattermost/mattermost-api-reference/blob/7287dc4fc9b556e7cec04f6753f5c3d7faba7f3d/v4/source/bots.yaml#L10